rules/do-you-create-a-sprint-forecast-aka-the-functionality-that-will-be-developed-during-the-sprint/rule.md

```diff
@@ -78,7 +78,8 @@ Figure: Good Example - copy this as email template and send to Product Owner
 
 More instructions are as below:
 
-1. Go to Azure DevOps and navigate to the current sprint's backlog view. E.g. https://dev.azure.com/ssw/Northwind/_sprints/backlog/Northwind%20Team/Northwind/Sprint%201
+1. Go to Azure DevOps and navigate to the current sprint's backlog view
+  E.g. **dev\.azure\.om/ssw/Northwind/\_sprints/backlog/Northwind%20Team/Northwind/Sprint%201**
```

Comment:
Please remove all the `\` from the url

Reply:
Fixed, replaced . by &#46; to avoid autolinks